src/forms-opportunities/handler.js

```diff
@@ -49,6 +50,66 @@ export async function formsAuditRunner(auditUrl, context) {
   };
 }
 
+export async function sendA11yIssuesToMystique(auditDataObject, scrapedData, context) {
+  const {
+    log, site,
+  } = context;
+
+  const auditData = JSON.parse(JSON.stringify(auditDataObject));
```

Comment:
can you try using getters on this object to extract siteId and auditId instead of converting it to JSON?